lib/App/index.js

```diff
@@ -311,6 +308,43 @@ class App {
 		this.debug(`Validated successfully`);
 		
 	}
+
+    _checkZwaveForSetting(driver, setting) {
+        if (setting.zwave &&
```

Comment:
What if a few properties are not provided?

Reply:
I could add validation errors for missing properties such as a size or index for zwave (as these are required at the very least)

Reply:
Sounds good.